test/gtest/coll_score/test_score_update.cc

```diff
@@ -34,7 +34,8 @@ UCC_TEST_F(test_score_update, non_overlap)
                UCC_COLL_TYPE_BARRIER);
     init_score(update, RLIST({RANGE(10, 20, 100), RANGE(30, 35, 1)}),
                UCC_COLL_TYPE_BARRIER);
-    EXPECT_EQ(UCC_OK, ucc_coll_score_update(score, update, 0, NULL, 0));
+    EXPECT_EQ(UCC_OK, ucc_coll_score_update(score, update, 0, NULL, 0,
+              UCC_COLL_TYPE_ALL));
```

Comment:
Will UCC_COLL_TYPE_ALL work for TL/SHM as well? Not all colls are supported there - will it fall back onto other TL?